packages/headless/src/api/generated-answer/generated-answer-client.ts

```diff
@@ -109,6 +109,7 @@ export class GeneratedAnswerAPIClient {
           Authorization: `Bearer ${accessToken}`,
           accept: '*/*',
         },
+        openWhenHidden: true,
```

Comment:
@erocheleau, iirc you have more context?

Reply:
A wild guess from me would be because IPX/Insight panel are displayed in an iframe of some sort (so, a different document); so when the end user focus out of the insight panel in Salesforce, the RGA answer stops streaming ?

Reply:
It's a weird behavior I noticed. I didn't try to reproduce specifically within Salesforce but it's pretty easy to reproduce within Chrome only. I am pretty sure it would do the same within Salesforce.

1. Start a search that returns a Qna response, 
2. when you see it loading change to a different chrome tab. 
3. Wait a few seconds, like enough for the Qna response to have finished generating 
4. and then come back to the first tab. 

The Qna response component is completely gone at this point. And I believe the component tries to re-start reading the stream, but at this moment it's now unavailable as it's already been accessed once. (Qna streams can only be consumed once). So you see a pretty 404 reply on the stream.

It's not a huge bug, and I haven't heard of a customer reporting this, but Louis found the cause pretty quickly so he fixed it :)